config/crd/kustomization.yaml

```diff
@@ -1,7 +1,7 @@
 commonLabels:
   cluster.x-k8s.io/v1alpha3: v1alpha3
   cluster.x-k8s.io/v1alpha4: v1alpha4
-  cluster.x-k8s.io/v1beta1: v1alpha5
+  cluster.x-k8s.io/v1beta1: v1alpha5_v1alpha6
```

Comment:
Our v1alpha4 API is also compatible with CAPI v1beta1, and was used as such in the v0.5 release. Should we also include this here?

Reply:
I'd say we must set this, yes. See the [Apply the contract version label](https://cluster-api.sigs.k8s.io/developer/providers/v1alpha2-to-v1alpha3.html#apply-the-contract-version-label-clusterx-k8sioversion-version1_version2_version3-to-your-crds) docs in the CAPI book

Reply:
We also have to update our [Readme](https://github.com/kubernetes-sigs/cluster-api-provider-openstack/#compatibility-with-cluster-api-and-kubernetes-versions) in this case